### DISCUSSION TITLE
demo.netbox.dev cable trace error

### DISCUSSION THREAD

**Discussion by devon-mar** [netbox-community/netbox#7173] [General]
I was playing around with v3.0 on `demo.netbox.dev` and it seems that the cable trace button is broken for devices from `netbox-demo-data`. I didn't open an issue because I'm not sure if this has to do with NetBox or the demo data. 

### Steps to reproduce:
1. Open the interface tab on one of the demo devices (eg. `dmi01-akron-sw01`)
2. Click the blue cable trace button on one of the interfaces
3. The following error occurs
```
<class 'KeyError'>

35

Python version: 3.8.10
NetBox version: 3.0.1
```

I also can't delete the cable and recreate it as I get a unique constraint violation:
```
<class 'django.db.utils.IntegrityError'>

duplicate key value violates unique constraint "dcim_cablepath_origin_type_id_origin_id_41b6f814_uniq"
DETAIL:  Key (origin_type_id, origin_id)=(27, 171) already exists.


Python version: 3.8.10
NetBox version: 3.0.1
```

**Comment by ryanmerolle** [comment]
raised as a bug in #7175
